src/rules.jl

```diff
@@ -58,10 +58,25 @@ will be hit as a fallback. This is the case for most rules.
 
 See also: [`rrule`](@ref), [`@scalar_rule`](@ref), [`RuleConfig`](@ref)
 """
-frule(::Any, ::Any, ::Vararg{Any}; kwargs...) = nothing
+frule(::Any, ::Any, ::Vararg{Any}) = nothing
```

Comment:
yes, we require at least 2 arguments.
The minimum number of arguments a function can have is zero.
Which has the case `frule(Tuple(), f)`.

Reply:
Either way is fine. The system doesn't care, but this makes it explicit that at least two arguments are expected, so you get a method error otherwise. I guess by the same reasoning, I should put back the two arguments below.